quickfixj-core/src/main/java/quickfix/mina/initiator/AbstractSocketInitiator.java

```diff
@@ -196,11 +207,20 @@ private void createSessions() throws ConfigError, FieldConvertError {
                 }
             }
         }
-        if (initiatorSessions.isEmpty()) {
-            throw new ConfigError("no initiators in settings");
-        }
         setSessions(initiatorSessions);
     }
+    
+    public void createDynamicSession(SessionID sessionID) throws ConfigError {
```

Comment:
Again for a new feature it would be nice to see tests. For a considerable feature like a dynamic session some of the end-to-end / regression tests seem appropriate.